src/transformers/models/xglm/configuration_xglm.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
+# Copyright The HuggingFace Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" XGLM model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+XGLM_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "facebook/xglm-564M": "https://huggingface.co/facebook/xglm-564M/resolve/main/config.json",
+    # See all XGLM models at https://huggingface.co/models?filter=xglm
+}
+
+
+class XGLMConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`~XGLMModel`]. It is used to instantiate an XGLM
```

Comment:
```suggestion
    This is the configuration class to store the configuration of a [`XGLMModel`]. It is used to instantiate an XGLM
```
Don't need the `~` anymore no cc @sgugger